integration_test/gce/gce_testing.go

```diff
@@ -670,7 +693,8 @@ var (
 // 'command' is what to run on the machine. Example: "cat /tmp/foo; echo hello"
 // 'stdin' is what to supply to the command on stdin. It is usually "".
 // TODO: Remove the stdin parameter, because it is hardly used and doesn't work
-//     on Windows.
+//
+//	on Windows.
```

Comment:
```suggestion
// on Windows.
```

Reply:
Changed in: [c5b2878](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/c5b28789a75d97d0c8419fa1350658b57a7e6e1f)